docs/styles.md

```diff
@@ -0,0 +1,51 @@
+# Styling content for RAISE
+
+In order to maintain consistency and reliability across all RAISE content, the styling associated with RAISE content is consolidated into a series of css classes located in `src/styles`. 
+
+RAISE Style classes are either written for specific elements, or can be applied to a variety of objects. Those differences are listed below. Content Developers 
+
+## Styling Classes 
+
+### Gray Box (os-raise-graybox)
+Add a Gray box around your element
+**Example**
```

Comment:
Similar comments on the markdown in this PR as the other as it relates to spelling and missing line breaks:
![image](https://user-images.githubusercontent.com/839025/196765089-36439d75-a0b1-4a71-b2e9-f8e6cafaad49.png)